lib/sdp/attribute.ex

```diff
@@ -3,29 +3,43 @@ defmodule Membrane.Protocol.SDP.Attribute do
   This module is responsible for parsing SDP Attributes.
   """
   alias __MODULE__.RTPMapping
+  use Bunch.Typespec
 
-  @valid_flags ["recvonly", "sendrecv", "sendonly", "inactive"]
-  @directly_assignable [
-    "cat",
-    "keywds",
-    "tool",
-    "orient",
-    "type",
-    "charset",
-    "sdplang",
-    "lang",
-    "rtpmap"
-  ]
-  @numeric ["ptime", "maxptime", "quality"]
+  @list_type flag_attributes :: [:recvonly, :sendrecv, :sendonly, :inactive]
+  @list_type value_attributes :: [
+               :cat,
+               :charset,
+               :keywds,
+               :orient,
+               :lang,
+               :rtpmap,
+               :sdplang,
+               :tool,
+               :type
+             ]
+
+  @list_type numeric_attributes :: [:maxptime, :ptime, :quality]
+
+  @type t ::
+          binary()
+          | flag_attributes
+          | {binary()
+             | value_attributes
+             | numeric_attributes
+             | :framerate, binary()}
+
+  @flag_values @flag_attributes |> Enum.map(&to_string/1)
+  @directly_assignable_values @value_attributes |> Enum.map(&to_string/1)
+  @numeric_values @numeric_attributes |> Enum.map(&to_string/1)
 
   @doc """
-  Parses SPD Attribute.
+  Parses SDP Attribute.
 
-  Value attributes formatted as `name:value` shall be parsed as `{name, value}`
-  other will be treated as property (flag) attributes.
-  Known attributes' names will be converted into atoms.
+  `t:value_attributes/0` and `t:numeric_attributes/0` formatted as `name:value`
+  shall be parsed as `{name, value}` other will be treated as
```

Comment:
_, while_ other will be treated...
I would also just use present simple here, but do as you wish